### PR TITLE
Fix specificity issues in label and in typography in general

### PR DESCRIPTION
**Pull request by hadasfa**
https://monday.monday.com/boards/3532714909/pulses/4949408216

### CODE REVIEW

File: src/components/Heading/Heading.module.scss [hunk 1]
```diff
@@ -2,9 +2,6 @@
 @import "../../styles/typography";
 @import "../../styles/global-css-settings";
 
-@include create-title-classes;
```
Comment (hadasfa) [inline src/components/Heading/Heading.module.scss:5]: not used

File: src/components/Label/Label.module.scss [hunk 2]
```diff
@@ -31,7 +31,6 @@
 }
 
 .kindFill.colorPrimary {
-  @include theme-prop(color, text-color-on-primary);
   @include theme-prop(background-color, primary-color);
```
Comment (hadasfa) [inline src/components/Label/Label.module.scss:34]: removed overrides everywhere we can set color by using the text color prop

File: src/components/Text/Text.tsx [hunk 3]
```diff
@@ -18,7 +20,7 @@ const Text: VibeComponent<TextProps, HTMLElement> = forwardRef(
     return (
       <Typography
         ref={ref}
-        className={cx(styles[size], styles[weight], className)}
+        className={cx(getStyle(styles, camelCase(size + "-" + weight)), className)}
```
Comment (hadasfa) [inline src/components/Text/Text.tsx:23]: replace 2 typography classes combination to one flat class

File: src/styles/typography.scss [hunk 4]
```diff
@@ -12,6 +13,32 @@ $p-empty-state-height: 16px;
 $h6-empty-state-height: 12px;
 $small-empty-state-height: 12px;
 
+@function contain($list, $value) {
```
Comment (hadasfa) [inline src/styles/typography.scss:16]: copy paste from monday ui styles, we will export the helper functions in different pr (less urgent)

**Review by SergeyRoyt — APPROVED** [review]

**Pull request merged**